azure-functions-maven-plugin/src/test/java/com/microsoft/azure/maven/function/invoker/storage/EventHubProcesser.java

```diff
@@ -134,7 +134,7 @@ private EventHubClient getEventHubClientByName(final String eventHubName) throws
                 .setEventHubName(eventHubName)
                 .setSasKeyName(SAS_KAY_NAME)
                 .setSasKey(getEventHubKey());
-            final EventHubClient eventHubClient = EventHubClient.create(connStr.toString(), executorService).get();
+            final EventHubClient eventHubClient = EventHubClient.createFromConnectionStringSync(connStr.toString(), executorService);
```

Comment:
does the `executorService` have to be `ScheduledExecutorService` ?

Reply:
yes. the SDK upgraded it so that we have to do like that.